docs/user/dql/functions.rst

```diff
@@ -1982,6 +1982,40 @@ Example::
     | True          | False         |
     +---------------+---------------+
 
+IF
+------
+
+Description
+>>>>>>>>>>>
+
+Specifications:
+
+1. IF(ES_TYPE0, ES_TYPE1, ES_TYPE2) -> ES_TYPE1 or ES_TYPE2
```

Comment:
Is there any reason we use ES_TYPE0/1/2? In my opinion, IF(expr, expr1, expr2) is more clear definition?